src/domain/app/header/Header.module.scss

```diff
@@ -0,0 +1,24 @@
+@import 'mixins';
+@import 'colors';
+
+$logo-width: 5em;
+$header-height: 5em;
+
+.header {
+  align-items: center;
+  border-bottom: 2px solid black;
+  display: flex;
+  padding: 10px;
+  height: $header-height;
+}
+
+.appName {
+  margin-left: 30px;
+}
+
+.logo {
+  @include helsinki-logo($hel-black);
+  background-position: center;
+  width: $logo-width;
+  height: inherit;
+}
```

Comment:
Wouldn't it be better to stick to the naming convention with other projects, which is starting `scss` files with small letters?

Reply:
Done